easybuild/easyconfigs/m/Mako/Mako-1.0.4-GCC-5.4.0-2.26-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['fed99dbe4d0ddb27a33ee4910d8708aca9ef1fe854e668387a9ab9a90cbf9059']
+
+dependencies = [('Python', '2.7.12', '-bare')]
+
+builddependencies = [('setuptools', '23.1.0', '-Python-2.7.12')]
```

Comment:
@vanzod please use `versionsuffix` here rather than hardcoding the Python version (or even using `%(version)s`)

```python
builddependencies = [('setuptools', '23.1.0', versionsuffix)]